inc/styles.php

```diff
@@ -243,18 +243,20 @@ static function row_style_fields( $fields ) {
 			'priority' => 16,
 		);
 
-		$fields['cell_alignment'] = array(
-			'name'     => __( 'Cell Vertical Alignment', 'siteorigin-panels' ),
-			'type'     => 'select',
-			'group'    => 'layout',
-			'options'  => array(
-				'flex-start' => __( 'Top', 'siteorigin-panels' ),
-				'center'     => __( 'Center', 'siteorigin-panels' ),
-				'flex-end'   => __( 'Bottom', 'siteorigin-panels' ),
-				'stretch'    => __( 'Stretch', 'siteorigin-panels' ),
-			),
-			'priority' => 17,
-		);
+		if ( siteorigin_panels_setting( 'legacy-layout' ) != 'always'  ) {
```

Comment:
Wouldn't you want to do the `SiteOrigin_Panels::is_legacy_browser()` check here too?

Reply:
No. This is what the person editing the page will see (it's when editing the row styles after all) rather than what the person on the frontend will see. The editor may have a really old browser on one computer and modern browser on another so to prevent possible issues when editing the page, I didn't check for this. That and it doesn't matter what the editor's browser can do - only the browser viewing the page on the frontend is what matters. 

Does that make sense? With that said, I can change it if you want but I don't think it's needed.

Reply:
Ah yeah, of course. Ok, all good. :+1: